librz/util/sdb/src/cdb_make.c

```diff
@@ -0,0 +1,196 @@
+// SPDX-FileCopyrightText: D. J. Bernstein <djb@cr.yp.to>
+// SPDX-License-Identifier: CC-PDDC
+
+#include <stdio.h>
+#include <stdlib.h>
+#include "sdb.h"
+#include "cdb.h"
+#include "cdb_make.h"
+#include "sdb_private.h"
+
+#if __APPLE__
```

Comment:
I am not sure if it makes sense to migrate these checks to Meson-based checks as well?

Reply:
i'd like to avoid touching the cdb code. i'd actually like in the future to just get the one from tinycdb.